### ISSUE TITLE
025-El Paso - Aging Problem

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#897]
In GitLab by @kdjstudios on May 11, 2018, 13:09

**Submitted by:**  "Juan Bernal" <juan.bernal@answernet.com>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/2018-05-10-33531/conversation

**Server:** Internal

**Client/Site:** 025

**Account:** E6692

**Issue:** 

We have an account in El Paso, Epicenter (T-025-E6692), who has a zero balance but is showing they owe $8.66 on our aging report. Is it possible to completely zero out this account and stop it from appearing in the aging report?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 11, 2018, 13:10

@smasih @pchaudhary @tim.traylor @thomas.gentry 

After looking into the account's account ledger it appears to be showing a negative balance there too; even though the invoices show a zero dollar balance. Please look into this and find the underlining problem.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 14, 2018, 08:28

@smasih @pchaudhary @tim.traylor @thomas.gentry 

Gary wrote:

> I am completely out of my mind seeing all of these account balance issues cropping up again.  On Thursday I want to hear how the heck this happened after spending all of that time before the merge getting the balances and A/R fixed.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on May 17, 2018, 07:04

Hi [Kyle] (@kdjstudios ), 

The mentioned account is showing correct account ledger after rejecting the transactions before the conversion date of the site. 

You can check the same on Merge server.

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 17, 2018, 09:54

@pchaudhary 

I just checked the El Paso aging report on MERGE and it is still showing the account with the $8.66. Looks like the issue reported is still not resolved. 

[aging__.pdf](/uploads/98bbb876214099137234846be8496c7f/aging__.pdf)

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on May 18, 2018, 24:12

Hi [Kyle] ( @kdjstudios ),  

> I just checked the El Paso aging report on MERGE and it is still showing the account with the $8.66. Looks like the issue reported is still not resolved.

* The new account ledger code which is rejecting the transactions held before & on the conversion date is currently attached to the account ledger page, that's the only reason due to which the aging report is still showing the wrong balance for EL Paso.

Once we will do it for the Aging report then it'll show the correct balance which is $ 0.00 currently.  

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 18, 2018, 08:25

@pchaudhary @tim.traylor @smasih @thomas.gentry 

I hope I am misunderstanding this; but are you stating that the aging was never working correctly, and is still not working?

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 21, 2018, 24:29

@kdjstudios 

> I hope I am misunderstanding this; but are you stating that the aging was never working correctly, and is still not working?


We have covered all the scenarios (identified till now) to prevent aging issue on the account **except one which is negative tax issue ("When a credit fee is marked as taxable" #833).**

And this was one of that scenario.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 23, 2018, 15:04

@smasih @pchaudhary @tim.traylor @thomas.gentry 

I wrote:

> The support team has informed us that this has been resolved. Please confirm everything is correct. Thank you. 

I would like to know how this was fixed? did we edit an invoice, alter a calculation, etc?

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 24, 2018, 24:52

@kdjstudios 



> I would like to know how this was fixed? did we edit an invoice, alter a calculation, etc?

This was edited because the root cause of this issue is still not fixed.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on May 24, 2018, 13:04

@smasih @pchaudhary @tim.traylor @thomas.gentry 

I am sorry but this is still confusing me. We are requesting how this was (temporarily resolved) and requested a specific explanation of what was completed. Please provide an explanation as to what was edited; not just that a manual edit was made. Vague responses help no one and only lead to further confusion and delay in resolution.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on May 25, 2018, 24:51

@kdjstudios 


System was showing that `T-025-E6692` owed $8.66.

Although the main account balance was $0 and it was clear that it's an aging issue. 

But, we fixed all the scenarios (which we were able to identify) leading to aging issue **except one, which was negative tax issue.**

So, on deleting or reverting a fee/invoice system distributes balance back to the previous invoices but in case of credit fee it leaves the tax amount out. 

This account was an example of that issue just like #833 

Thomas is working on this (as far as I know from the comments) so the fix is not deployed yet that is why we made a manual change and deleted $8.66 from the balance section of the faulty invoice.
